api/api/models/hack.py

```diff
@@ -1,9 +1,27 @@
 from django.db import models
-from api.models import Hackathon, JudgingExpo, JudgingCriteria
+from api.models import Hackathon
+from api.models.judging_criteria import JudgingCriteria
+from api.models.judging_expo import JudgingExpo
```

Comment:
This whole thing was done cause the importing was out of order in api.models.__init__.py